include/cpp2util.h

```diff
@@ -197,7 +197,9 @@
         #endif
         #include <stdexcept>
         #if __has_include(<stdfloat>)
-            #include <stdfloat>
+            #if (defined(_MSC_VER) && _HAS_CXX23) || !defined(_MSC_VER)
```

Comment:
This check can be simpler:

    #if !defined(_MSC_VER) || _HAS_CXX23

Reply:
Thanks, updated!